flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/shuffle/TieredResultPartition.java

```diff
@@ -63,6 +67,8 @@ public class TieredResultPartition extends ResultPartition {
 
     private final TieredStorageResourceRegistry tieredStorageResourceRegistry;
 
+    private final TieredStorageNettyService nettyService;
```

Comment:
Let's change this to `TieredStorageNettyServiceImpl` to make it explicit that the result partition sees the actual netty service implementation.